suite/runsuite.cmake

```diff
@@ -546,6 +550,30 @@ if (ARCH_IS_X86 AND UNIX AND (a64_on_x86_only OR NOT arg_automated_ci))
     " OFF ${arg_package} "")
 endif ()
 
+if (ARCH_IS_X86 AND UNIX AND cross_riscvxx_linux_only)
+# Do riscv cross-compilation if cross_riscvxx_linux_only is on.
+  set(orig_extra_ctest_args ${extra_ctest_args})
+  set(run_tests OFF)
```

Comment:
Please add something like `TODO i#3544: Run tests under QEMU` before this line as the current line is not what we want long-term.

Reply:
Added it.